### DISCUSSION TITLE
Questions about the brats2020 data set

### DISCUSSION THREAD

**Discussion by nebstudio** [MIC-DKFZ/nnUNet#886] [Q&A] [answered]
Hello, I am sorry to trouble you again, I am using nnUNet's nnUNetTrainerV2BraTSRegions_DA4_BN_BD for training, but the result I got is very low, some Dice is 0.6, I want to ask you to answer that it needs to be processed to make the result tend to the article The result?

**Comment by FabianIsensee** [comment]
Can you be more specific? what Dice is low? All regions?

**Reply by nebstudio** [reply]
![image](https://user-images.githubusercontent.com/60005228/145849976-368258fd-64e0-4f86-918e-879fd7b5b9e6.png)
Not all, the results I saw in the summary.json file in the validation_raw_postprocessed folder are on this picture, should I look at this file? Forgive me for constantly asking questions, nnUNet is great, I want to clarify it, and also want to ask if your Dice after each epoch is the result of the verification set?

**Comment by FabianIsensee** [comment, marked as answer]
The 'Dice' after each epoch is no Dice. It even prints that in the log every time an epoch ends. This is just a substitute. Please take a look at the documentation where this is explained.
The summary.json is NOT where you need to look for the BraTS performance. nnU-Net does not know how to evaluate the regions. You need to evaluate with the `evaluate_BraTS_folder` function, located in the BraTS 2020 dataset conversion file
